.github/workflows/ci.yml

```
@@ -24,6 +24,23 @@ jobs:
+          envkey_PROD_TURSO_AUTH_TOKEN: ${{ secrets.PROD_TURSO_AUTH_TOKEN }}
+          envkey_PROD_DEFAULT_SUPER_ADMIN_USERNAME: ${{ secrets.PROD_DEFAULT_SUPER_ADMIN_USERNAME }}
+          envkey_PROD_DEFAULT_SUPER_ADMIN_PASSWORD: ${{ secrets.PROD_DEFAULT_SUPER_ADMIN_PASSWORD }}
+
+          envkey_DEV_TURSO_DATABASE_URL: ${{ secrets.DEV_TURSO_DATABASE_URL }}
+          envkey_DEV_TURSO_AUTH_TOKEN: ${{ secrets.DEV_TURSO_AUTH_TOKEN }}
+          envkey_DEV_DEFAULT_SUPER_ADMIN_USERNAME: ${{ secrets.DEV_DEFAULT_SUPER_ADMIN_USERNAME }}
+          envkey_DEV_DEFAULT_SUPER_ADMIN_PASSWORD: ${{ secrets.DEV_DEFAULT_SUPER_ADMIN_PASSWORD }}
+
+          envkey_TEST_TURSO_DATABASE_URL: ${{ secrets.TEST_TURSO_DATABASE_URL }}
+          envkey_TEST_DEFAULT_SUPER_ADMIN_USERNAME: ${{ secrets.TEST_DEFAULT_SUPER_ADMIN_USERNAME }}
+          envkey_TEST_DEFAULT_SUPER_ADMIN_PASSWORD: ${{ secrets.TEST_DEFAULT_SUPER_ADMIN_PASSWORD }}        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)